virttest/qemu_storage.py

```diff
@@ -828,23 +828,26 @@ def compare_to(self, target_image, source_cache_mode=None,
 
         return result
 
-    def check_image(self, params, root_dir, force_share=False):
+    def check_image(self, params, root_dir, force_share=False, output=None):
         """
         Check an image using the appropriate tools for each virt backend.
 
         :param params: Dictionary containing the test parameters.
         :param root_dir: Base directory for relative filenames.
+        :param output: The format of the output.
 
         :note: params should contain:
                image_name -- the name of the image file, without extension
                image_format -- the format of the image (qcow2, raw etc)
 
         :raise VMImageCheckError: In case qemu-img check fails on the image.
+        :return: The output of check result if the image exists, or ''.
         """
         image_filename = self.image_filename
         logging.debug("Checking image file %s", image_filename)
         image_is_checkable = self.image_format in ['qcow2', 'qed']
         force_share &= self.cap_force_share
+        cmd_result = ''
```

Comment:
why assign an empty string to `cmd_result`?

Reply:
For the consistency of `if` and `else`. `cmd_result` is assigned only in `if` before.

Reply:
Python will simply return `None` if there is no return in function, this is sufficient.